.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: php
 php:
   - 5.6
+  - 7.0
+  - 7.1
```

Comment:
Maybe adding `7.2` and `nightly` would allow the lib to be more prepared for the new PHP versions (at least `nightly` should be as `allowed_failure` because travis sometimes does some weird stuff there)